feat(docs): Add aztec-wallet proving (#10847)
AztecProtocol/aztec-packages: docs/docs/guides/developer_guides/local_env/sandbox_proving.md

The sandbox will also deploy 3 Schnorr account contracts on startup. The sandbox will need to generate transaction proofs for deployment, which will take additional time.

Once everything has been set up, you will see that the PXE is listening on `localhost:8080` as you would see with the sandbox running in the default mode. At this point you can use the sandbox as you would without client-side proving enabled.

## Proving with `aztec-wallet`

> Alternatively, you can enable proving for transactions using `aztec-wallet` and not all transactions going to the sandbox.

Reviewer: ```suggestion
Alternatively, you can enable proving on a per-transaction basis using the `aztec-wallet` CLI
```

Open `~/.aztec/bin/aztec-wallet` and update the `ENV_VARS_TO_INJECT` variable to:

```bash
# ~/.aztec/bin/aztec-wallet
export ENV_VARS_TO_INJECT="WALLET_DATA_DIRECTORY SSH_AUTH_SOCK BB_BINARY_PATH PXE_PROVER_ENABLED BB_WORKING_DIRECTORY"
```

Export the following envnironment variables in the terminal where you will run `aztec-wallet` commands:

```bash
export BB_BINARY_PATH=/usr/src/barretenberg/cpp/build/bin/bb
export PXE_PROVER_ENABLED=1
export BB_WORKING_DIRECTORY=~/bb-temp
```

> Now send transactions from `aztec-wallet`, and proving will be enabled.

Reviewer: can we add a reference to how to send tx using aztec-wallet? hyperlink or a dummy example
